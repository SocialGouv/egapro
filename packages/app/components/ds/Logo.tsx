import React, { FunctionComponent } from "react"
import { Image } from "@chakra-ui/react"

export type LogoProps = Record<string, unknown>

const Logo: FunctionComponent<LogoProps> = () => {
  return (
    <Image
      src={"/consulter-index/icons/marianne.svg"}
      alt="Aller à la page d'accueil du Ministère du travail, de l'emploi et de l'insertion"
      width={88}
    />
  )
}

export default Logo
