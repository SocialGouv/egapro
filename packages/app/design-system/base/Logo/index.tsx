import React, { FunctionComponent } from "react"

export type LogoProps = {}

const Logo: FunctionComponent<LogoProps> = () => {
  return (
    <p className="fr-logo">
      Ministère <br />
      du Travail,
      <br /> de l’Emploi
      <br /> et de l’Insertion
    </p>
  )
}

export default Logo
