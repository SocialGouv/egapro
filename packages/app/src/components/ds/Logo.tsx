import { Image } from "@chakra-ui/react";
import type { FC } from "react";
import React from "react";

export type LogoProps = Record<string, unknown>;

export const Logo: FC<LogoProps> = () => {
  return (
    <Image
      src={"/consulter-index/icons/marianne.svg"}
      alt="Aller à la page d'accueil du Ministère du travail, de l'emploi et de l'insertion"
      width={88}
    />
  );
};
