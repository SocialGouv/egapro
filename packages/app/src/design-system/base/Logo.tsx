export const Logo = () => {
  return (
    <p className="fr-logo">
      Ministère <br />
      du Travail,
      <br /> de l’Emploi
      <br /> et de l’Insertion
    </p>
  );
};
