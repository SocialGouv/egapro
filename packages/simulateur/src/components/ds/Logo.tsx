import React, { FunctionComponent } from "react"
import { Image } from "@chakra-ui/react"

export type LogoProps = Record<string, unknown>

const Logo: FunctionComponent<LogoProps> = () => {
  return (
    <Image
      src={process.env.PUBLIC_URL + "/marianne.svg"}
      alt="Aller à la page d'accueil du Ministère du travail, de l'emploi et de l'insertion"
      width={88}
    />
  )
}

export default Logo
