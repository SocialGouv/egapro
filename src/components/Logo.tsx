/** @jsx jsx */
import { css, jsx } from "@emotion/core"

function Logo({ layout = "default" }: { layout?: "default" | "mobile" }) {
  return (
    <div css={[styles.container, layout === "mobile" && styles.containerMobile]}>
      <img
        css={[styles.image, layout === "mobile" && styles.imageMobile]}
        src={process.env.PUBLIC_URL + "/marianne.svg"}
        alt="Aller à la page d'accueil du Ministère du travail, de l'emploi et de l'insertion"
      />
    </div>
  )
}

const styles = {
  container: css({
    flexShrink: 0,
    display: "flex",
    flexDirection: "column",
    width: 76,
  }),
  containerMobile: css({
    flexDirection: "row",
    alignItems: "flex-end",
    width: "auto",
  }),
  image: css({
    display: "block",
    width: 90,
  }),
  imageMobile: css({
    width: 60,
    height: 36,
    marginRight: 3,
  }),
  text: css({
    marginTop: 3,
    fontSize: 9,
    textAlign: "center",
  }),
}

export default Logo
