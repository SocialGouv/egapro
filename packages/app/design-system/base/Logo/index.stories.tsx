import React from "react"
import { ComponentStory, ComponentMeta } from "@storybook/react"

import Logo from "./index"

export default {
  title: "Base/Logo",
  component: Logo,
} as ComponentMeta<typeof Logo>

const Template: ComponentStory<typeof Logo> = (args) => <Logo {...args} />

export const Default = Template.bind({})
Default.args = {}
