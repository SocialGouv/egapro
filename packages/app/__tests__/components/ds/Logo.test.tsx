import { render } from "@testing-library/react"
import Logo from "@/components/ds/Logo"

test("should match snapshot", () => {
  const view = render(<Logo />)
  expect(view).toMatchSnapshot()
})
