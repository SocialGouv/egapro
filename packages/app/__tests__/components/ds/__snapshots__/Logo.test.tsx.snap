// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div
      id="chakra-toast-portal"
    >
      <ul
        id="chakra-toast-manager-top"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
      />
      <ul
        id="chakra-toast-manager-top-left"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-top-right"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-bottom-left"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-bottom"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
      />
      <ul
        id="chakra-toast-manager-bottom-right"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
    </div>
    <div>
      <img
        alt="Aller à la page d'accueil du Ministère du travail, de l'emploi et de l'insertion"
        class="chakra-image css-6t4orf"
        src="/consulter-index/icons/marianne.svg"
      />
    </div>
  </body>,
  "container": <div>
    <img
      alt="Aller à la page d'accueil du Ministère du travail, de l'emploi et de l'insertion"
      class="chakra-image css-6t4orf"
      src="/consulter-index/icons/marianne.svg"
    />
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
